status/format.go

```diff
@@ -196,3 +201,17 @@ func FormatFootnotes(taskRuns []*helpers.TaskRun) (string, error) {
 	}
 	return strings.Join(footnotes, "\n"), nil
 }
+
+// FormatPipelineURL accepts a TaskRun and returns a Markdown friendly representation of its test suite, if any.
```

Comment:
The description of this function needs to be updated

Reply:
Updated